### ISSUE TITLE
WCAG 2.1 2.5.2: single atomic rule or multiple atomics with a composite?

### ISSUE THREAD

**Issue by patrickhlauke** [act-rules/act-rules.github.io#1053]
For some initial internal work (which I'll also be submitting here for refinement/another pair of eyes and brains) I'm trying to tackle some of the "fluffier" (read: more difficult to write up in a more methodical/technical way) SCs.

Currently, I'm looking at 2.5.2 Pointer Cancellation https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html#dfn-single-pointer and am wondering: what would make most sense? A single atomic rule that, under expectations, has the "at least one of the following is true:" and then the three bullet points (and moving the "Essential" clause under the "Assumptions" for the rule)? Or painfully setting up 3 or even 4 separate atomic rules for each of the first three bullets (and splitting the "Abort or Undo" into two separate atomic rules covering the slightly different aspects there) and then having one composite rule with those as inputs and codifying there that if at least one passes, then the composite rule required for conformance passes?

Thoughts/suggestions/"we've tried this approach"/"let's collaborate on this" would be welcome :)

**Comment by carlosapaduarte** [comment]
I would go with a composite with multiple atomic rules.
Per the ACT rules format "When a test target meets all expectations, the test target passed the rule. If the test target does not meet all expectations, the test target failed the rule."
Having an expectation with "at least one of the following is true" would go against this intent.

**Comment by patrickhlauke** [comment]
yeah, had a feeling that was the case. thanks for confirming.
